card/spec/lib/card/reference_spec.rb

```diff
@@ -55,9 +55,9 @@
   end
 
   it "container inclusion" do
-    Card.create :name=>'bob+city'
-    Card.create :name=>'address+*right+*default',:content=>"{{_L+city}}"
-    Card.create :name=>'bob+address'
+    Card.create name: 'bob+city'
+    Card.create name: 'address+*right+*default',content: "{{_L+city}}"
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.